crates/pop-fork/src/models.rs

```diff
@@ -0,0 +1,31 @@
+use crate::schema::{blocks, storage};
+use diesel::{Insertable, Queryable, Selectable};
+
+#[derive(Insertable, Clone)]
+#[diesel(table_name = storage)]
+pub struct NewStorageRow<'a> {
```

Comment:
Could we change visibility to `pub(crate)` instead ? Do we need to expose this at this time ?